qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -78,20 +78,32 @@ def run(self, qobj):
 
         return job
 
-    def properties(self, refresh=False):
-        """Return the online backend properties.
+    def properties(self, refresh=False, datetime_filter=None):
```

Comment:
After giving it some thought, can you rename the parameter to just `datetime`, for user-clarity?